src/ota/ota_handler.rs

```diff
@@ -36,11 +36,12 @@ use crate::ota::rauc::OTARauc;
 use crate::ota::OtaError;
 use crate::ota::{Ota, OtaId, OtaStatus};
 use crate::repository::file_state_repository::FileStateRepository;
-use crate::MAX_OTA_OPERATION;
 
 use super::event::{OtaOperation, OtaRequest};
 use super::PersistentState;
 
+const MAX_OTA_OPERATION: usize = 2;
```

Comment:
Why do you redefine `MAX_OTA_OPERATION`?

Reply:
I moved it from the main lib module to the OTA module since it makes more sense to have it in the same place where we're using it. Given that it could be disabled, it avoids unused warnings when the ̀zbus̀ feature is disabled.